docs/development/dev_box_de.md

```diff
@@ -49,4 +50,18 @@ Zum Entschlüssen kann folgender Befehl verwendet werden:
 gpg --decrypt .vagrant.rb.asc > .vagrant.rb
 ```
 
-> **Hinweis:** Bei Änderungen in der `.vagrant.rb` muss diese erneut verschlüsselt und anschließend gelöscht werden! 
\ No newline at end of file
+> **Hinweis:** Bei Änderungen in der `.vagrant.rb` muss diese erneut verschlüsselt und anschließend gelöscht werden!
+
+### Zertifikate
+Standardmäßig erstellt das Setup des CES ein selbstsigniertes SSL-Zertifikat für die Absicherung der HTTPS-Verbindungen.
+Das hat den Nachteil, dass Browser diesem Zertifikat nicht vertrauen und dafür Ausnahmen im Browser eingerichtet werden müssen.
+Um das zu vermeiden, kann für die Entwicklung beim Erstellen der Dev-Box ein Zertifikat verwendet werden, das mit dem Tool [mkcert](https://github.com/FiloSottile/mkcert) erstellt wird. 
+Diesem Zertifikat wird lokal auf dem Entwicklungs-Rechner vertraut.
+
+Nachdem `mkcert`[installiert](https://github.com/FiloSottile/mkcert#installation) wurde, muss es einmalig mit folgendem Befehl initialisiert werden:
+```shell
+mkcert -install
+```
+
+Anschließend kann in der [Konfiguration](#konfiguration) der Wert für `certificate_type` auf `mkcert` gesetzt werden.
+Im Setup wird nun ein con `mkcert` erstelltes Zertifikat verwendet.
```

Comment:
The certificate will be created, if it doesn't exists, too.

and "con" -> "von"